### PR TITLE
Java 11

### LINKED ISSUE #353: Move to Java 11
As discussed, we will update to Java 11 as a minimum requirement with version 8.
Within this step, we can build true JMS compatible modules without Maven hackery.

### PR DESCRIPTION
**Pull request by fh-ms**
See #353

### CODE REVIEW

File: cache/hibernate/src/main/java/one/microstream/cache/hibernate/types/CacheRegionFactory.java [hunk 1]
```diff
@@ -58,10 +58,10 @@ public class CacheRegionFactory extends RegionFactoryTemplate
 {
 	private final    CacheKeysFactory                   cacheKeysFactory;
 	private volatile CacheManager                       cacheManager;
-	private AtomicBoolean                               isExplicitCacheManager = new AtomicBoolean();
+	private final AtomicBoolean                               isExplicitCacheManager = new AtomicBoolean();
```
Comment (zdenek-jonas) [inline cache/hibernate/src/main/java/one/microstream/cache/hibernate/types/CacheRegionFactory.java]: Format?

File: cache/hibernate/src/main/java/one/microstream/cache/hibernate/types/CacheRegionFactory.java [hunk 2]
```diff
@@ -58,10 +58,10 @@ public class CacheRegionFactory extends RegionFactoryTemplate
...
 	private volatile CacheConfiguration<Object, Object> cacheConfiguration;
 	private volatile MissingCacheStrategy               missingCacheStrategy;
-	private AtomicLong                                  cacheLockTimeout = new AtomicLong();
+	private final AtomicLong                                  cacheLockTimeout = new AtomicLong();
```
Comment (zdenek-jonas) [inline cache/hibernate/src/main/java/one/microstream/cache/hibernate/types/CacheRegionFactory.java]: Format

File: docs/modules/intro/pages/system-requirements.adoc [hunk 3]
```diff
@@ -8,37 +8,37 @@ Tested and officially supported JDKs:
...
 | https://github.com/ojdkbuild/ojdkbuild[ojdkbuild]
-| 8, 11
+| 11
 
 | https://www.microsoft.com/openjdk/[MicroSoft OpenJDK]
 | 11, 17
 
 | http://dragonwell-jdk.io/[Alibaba Dragonwell]
-| 8, 11
+| 11
```
Comment (zdenek-jonas) [inline docs/modules/intro/pages/system-requirements.adoc]: 17 is also tested and works, it could be added

**Review by zdenek-jonas — APPROVED** [review]
Please check the comments, otherwise, I agree.

File: cache/hibernate/src/main/java/one/microstream/cache/hibernate/types/CacheRegionFactory.java (hunk 1 above)
Comment (fh-ms) [reply cache/hibernate/src/main/java/one/microstream/cache/hibernate/types/CacheRegionFactory.java]: Fixed

File: cache/hibernate/src/main/java/one/microstream/cache/hibernate/types/CacheRegionFactory.java (hunk 2 above)
Comment (fh-ms) [reply cache/hibernate/src/main/java/one/microstream/cache/hibernate/types/CacheRegionFactory.java]: Fixed

File: docs/modules/intro/pages/system-requirements.adoc (hunk 3 above)
Comment (fh-ms) [reply docs/modules/intro/pages/system-requirements.adoc]: Added

**Pull request merged**